Doppler.Sap.Test/CreditNoteHandlerTest.cs

```diff
@@ -0,0 +1,217 @@
+using System;
+using System.Collections.Generic;
+using System.Net;
+using System.Net.Http;
+using System.Threading;
+using System.Threading.Tasks;
+using Doppler.Sap.Factory;
+using Doppler.Sap.Mappers.Billing;
+using Doppler.Sap.Models;
+using Doppler.Sap.Services;
+using Doppler.Sap.Utils;
+using Doppler.Sap.Validations.Billing;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+using Moq;
+using Moq.Protected;
+using Xunit;
+
+namespace Doppler.Sap.Test
+{
+    public class CreditNoteHandlerTest
+    {
+        [Fact]
+        public async Task CreditNoteHandler_ShouldBeCreateCreditNoteInSap_WhenQueueHasOneValidElement()
+        {
+            var sapSaleOrderInvoiceResponse = new SapSaleOrderInvoiceResponse
+            {
+                BillingSystemId = 2,
+                CardCode = "CD001",
+                DocumentLines = new List<SapDocumentLineResponse>()
+            };
+
+            var billingValidations = new List<IBillingValidation>
```

Comment:
This variable is not used, you can remove it

Reply:
Done